### PR TITLE
fix karmada icon on dark theme

### PR DESCRIPTION
**Pull request by Arhell**
**What type of PR is this?**

/kind bug
/kind design

**Which issue(s) this PR fixes**:
reolve https://github.com/karmada-io/website/issues/76

### CODE REVIEW

**Review by rgrupesh — COMMENTED** [review]
Checked deployment. Looks good!

**Comment by Arhell** [comment]
/assign @RainbowMango
https://github.com/karmada-io/website/pull/80#pullrequestreview-998196161

**Comment by Arhell** [comment]
@RainbowMango
Thanks, done

**Comment by RainbowMango** [comment]
Seems the icon is different from the https://github.com/cncf/artwork/blob/master/projects/karmada/icon/white/karmada-icon-white.svg.

Did you edit it somehow?

File: docusaurus.config.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ module.exports = {
       logo: {
         alt: "Karmada",
         src: "img/logo.svg",
-        srcDark: "img/logoDark.svg",
+        srcDark: "img/logoWhite.svg",
```
Comment (RainbowMango) [inline docusaurus.config.js:39]:
```suggestion
        srcDark: "img/artwork/karmada-icon-white.svg",
```
I'd prefer to put this image in a separate directory and rename it.

**Comment by Arhell** [comment]
> Seems the icon is different from the https://github.com/cncf/artwork/blob/master/projects/karmada/icon/white/karmada-icon-white.svg.
> 
> Did you edit it somehow?

no, downloaded and put in our repository

**Comment by RainbowMango** [comment]
> no, downloaded and put in our repository

I don't know how you download it, but they are different from my side.
The file in the artwork repo is `1` line.

**Comment by Arhell** [comment]
> > no, downloaded and put in our repository
> 
> I don't know how you download it, but they are different from my side. The file in the artwork repo is `1` line.

https://raw.githubusercontent.com/cncf/artwork/master/projects/karmada/icon/white/karmada-icon-white.svg

and save as 
they are identical

**Comment by Arhell** [comment]
https://github.com/karmada-io/website/pull/80/files#diff-3c530b4f77589135d56b7cf8c7ddb47e28f264f6440636f705f491c4fecec79a

**Pull request merged**